dayrepo/snippets/views.py

```diff
@@ -154,3 +155,31 @@ def post(self, request):
 
 
 snippet_post = SnippetView.as_view()
+
+# checklist投稿機能
+class ChecklistView(View):
+    # 新規入力画面
+    def get(self, request):
+        # 投稿ボタンで投稿ページへ
+        return render(
+            request,
+            "snippet_checklist.html",
+            {
+                "form": ChecklistForm,
+            },
+        )
+
+    # 投稿機能
+    def post(self, request):
+
+        # Checklist Form
+        checklist = ChecklistForm(request.POST)
+        # ※Process formとのbool入力処理の違い
+        # Process formではboolの入力値が"on"/"off"になっていたが、
+        # formの引数にrequest.POSTを指定した場合、"True"/"False"で
+        # 登録してくれる(結論、変換しなくていい)
```

Comment:
たぶんまだ私も読むことがありそうなのと、意味合いとしてちょっと違う箇所の見つけたのでこんなんでどう?
```suggestion
        # 【Process form との bool 入力処理の違い】
        # チェックボックス(bool)のリクエスト値は
        # 内部的に "on" か "off" で受け取っていた。
        # Process form 側では if 文による True/False への変換を行ったが
        # form クラスの引数に request.POST を渡す場合、
        # 変換しなくてもいい感じに bool で登録してくれる
```

気になったところ
> Process formではboolの入力値が"on"/"off"になっていたが、

とあり、process 側と checklist 側で入力値が違っていそうな記述があるが、今回の checklist でも入力値自体は同じく "on"/"off" になっていたため、意味としておかしい